[OSDOCS#18693]: CQA Remediation for Cert Manager Operator OSDOCS-17173 - PR3 (#116412)
openshift/openshift-docs: modules/cert-manager-operator-release-notes-1-19-0.adoc

// Module included in the following assemblies:
//
// * security/cert_manager_operator/cert-manager-operator-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="cert-manager-operator-release-notes-1-19-0_{context}"]
= {cert-manager-operator} 1.19.0

> Issued: 20 April 2026

Reviewer: ```suggestion

```

[role="_abstract"]

> The following advisories are available for the {cert-manager-operator} 1.19.0:

Reviewer: ```suggestion
 Review the release notes for the {cert-manager-operator} 1.9.0 to learn what is new and updated with this release.

Issued: 20 April 2026

The following advisories are available for the {cert-manager-operator} for {product-title} 1.9.0: 
```
This is a better short description. Short descriptions cannot end in colons.

* link:https://access.redhat.com/errata/RHBA-2026:9025[RHBA-2026:9025]
* link:https://access.redhat.com/errata/RHBA-2026:8956[RHBA-2026:8956]

Version `v1.19.4` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.19.4`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.19#v1194[cert-manager project release notes for v1.19.4].

[id="cert-manager-operator-1-19-0-features-enhancements_{context}"]
== New features and enhancements

Distribution of trust bundles with the trust manager operand (Technology Preview)::
In this release, the {cert-manager-operator} adds support for the trust-manager operand as a Technology Preview feature. You can now install the trust-manager operand to automate the secure distribution of trust bundles, such as certificate authority (CA) certificates, to application namespaces across your cluster. For more information, see xref:../../security/cert_manager_operator/cert-manager-trust-manager.adoc#cert-manager-trust-manager[Distributing certificates by using trust-manager operand].

Support for configuring the certificate request backoff duration::

> In this release, the {cert-manager-operator} adds support for the `--certificate-request-minimum-backoff-duration` flag. With this flag, you can configure the minimum backoff period for certificate requests by override the default configuration. For more information, see xref:../../security/cert_manager_operator/cert-manager-customizing-api-fields.adoc#cert-manager-overridable-arguments_cert-manager-customizing-api-fields[Overridable arguments for the cert-manager components].

Reviewer: ```suggestion
In this release, the {cert-manager-operator} adds support for the `--certificate-request-minimum-backoff-duration` flag. With this flag, you can configure the minimum backoff period for certificate requests by overriding the default configuration. For more information, see xref:../../security/cert_manager_operator/cert-manager-customizing-api-fields.adoc#cert-manager-overridable-arguments_cert-manager-customizing-api-fields[Overridable arguments for the cert-manager components].
```

> [id="cert-manager-operator-1-19-0-known-issues_{context}"]

Reviewer: ```suggestion
[id="cert-manager-operator-1-19-0-fixed-issues_{context}"]
```
